src/NServiceBus.Metrics.Tests/NServiceBus.Metrics.approved.cs

```diff
@@ -5,6 +5,19 @@
 namespace NServiceBus
 {
     
+    public interface IDurationProbe : NServiceBus.IProbe
+    {
+        void Register(System.Action<System.TimeSpan> observer);
+    }
+    public interface IProbe
+    {
+        string Description { get; }
+        string Name { get; }
+    }
+    public interface ISignalProbe : NServiceBus.IProbe
+    {
+        void Register(System.Action observer);
```

Comment:
If this is called Signal probe what is the signal then since action takes no input